app/src/main/java/org/sopt/daangnmarket_android/ui/view/write/WriteActivity.kt

```diff
@@ -1,12 +1,34 @@
 package org.sopt.daangnmarket_android.ui.view.write
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.Toast
 import org.sopt.daangnmarket_android.R
+import org.sopt.daangnmarket_android.databinding.ActivityWriteBinding
+import org.sopt.daangnmarket_android.ui.view.main.MainActivity
 
 class WriteActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityWriteBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        binding = ActivityWriteBinding.inflate(layoutInflater)
         setContentView(R.layout.activity_write)
+        //버튼을 눌러도 아무일이 일어나지 않고 있습니다......... 혹시 제가 바보같은 짓을 했나요?
```

Comment:
```suggestion
        binding = ActivityWriteBinding.inflate(layoutInflater)
        setContentView(binding.root)
        //버튼을 눌러도 아무일이 일어나지 않고 있습니다......... 혹시 제가 바보같은 짓을 했나요?
```

`setContentView()` 함수는 화면에 View를 보여주는 함수인데, 여기에 binding 객체의 root를 넣어야 우리가 16번 코드에서 초기화한 binding 객체를 사용해서 View를 다룰 수 있습니다. 조금 더 자세하게 이야기해보자면,

``` kotlin
binding = ActivityWriteBinding.inflate(layoutInflater)
```
activity_write.xml에서 구현한 View에 대한 binding 객체를 inflate, 즉 xml 코드를 Java 코드로 변환시키는 코드입니다.

``` kotlin
setContentView(binding.root)
```
초기화한 binding 객체를 사용하여 View를 화면에 띄웁니다.

기존의 코드에서는 binding 객체를 사용해 View를 띄운 것이 아닙니다. 그래서 binding이라는 객체 변수를 통해 View에 접근할 수 없던 것입니다.

Reply:
이해됐습니다..! 자세한 설명 감사합니다 👍🏻

Reply:
머싯네요 👍